libs/shared/src/lib/components/markdown/components/markdown-editor/classes/bold-button.class.ts

```diff
@@ -0,0 +1,19 @@
+import { MarkdownSyntaxEnum } from '../enums';
+import { MarkdownButton } from '../interfaces/markdown-button.interface';
+
+export class BoldButtonClass implements MarkdownButton {
+    public readonly markdownSyntaxEnum = MarkdownSyntaxEnum.Bold;
+    public readonly matIconName = 'format_italic';
+
+    public getMarkdownSyntax(markdownText: string): string {
+        const trimmedMarkdownText = markdownText.trim();
+
+        const MARKDOWN_MAP = new Map<string, string>([
+            [MarkdownSyntaxEnum.Bold, `**${trimmedMarkdownText}**`],
+            [MarkdownSyntaxEnum.Italic, `_${trimmedMarkdownText}_`],
+            [MarkdownSyntaxEnum.Header, `# ${trimmedMarkdownText}`]
+        ]);
+
+        return MARKDOWN_MAP.get(this.markdownSyntaxEnum)!;
```

Comment:
```suggestion
        return `**${markdownText.trim()}**`;
```

Замість всього що є в методі і так в кожному відповідному класі